other/nightly/install.sh

```
@@ -288,169 +431,245 @@ if [ -x "$(command -v snap)" ]; then
+        systemctl enable docker.service >/dev/null 2>&1
+        if ! [ -x "$(command -v docker)" ]; then
+            echo " - Failed to install Docker with pacman. Try to install it manually."
+            echo "   Please visit https://wiki.archlinux.org/title/docker for more information."
+            exit 1
+        fi
+        ;;
+    "amzn")
+        dnf install docker -y >/dev/null 2>&1
+        DOCKER_CONFIG=${DOCKER_CONFIG:-/usr/local/lib/docker}
+        mkdir -p $DOCKER_CONFIG/cli-plugins >/dev/null 2>&1
+        curl -sL https://github.com/docker/compose/releases/latest/download/docker-compose-$(uname -s)-$(uname -m) -o $DOCKER_CONFIG/cli-plugins/docker-compose >/dev/null 2>&1
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)